crypto/rand/rand_lib.c

```diff
@@ -163,6 +163,11 @@ size_t rand_drbg_get_entropy(RAND_DRBG *drbg,
 
         if (buffer != NULL) {
             size_t bytes = 0;
+            RAND_DRBG *master_drbg = RAND_DRBG_get0_master();
+
+            if (drbg->parent == master_drbg) {
+                 prediction_resistance = 1;
```

Comment:
why only for the master?

Reply:
Lab request - needed only for master.

"FIPS 140-3 IG D.K Resolution 5. Requires that the master DRBG shall have prediction resistance enabled, or shall be reseeded before requesting an output."

Can't enable prediction resistance as it's not implemented, (hence the #if 0 above) but setting the variable prediction_resistance = 1 when it's the master DRBG forces reseed.